_data/toc.yaml

```diff
@@ -511,6 +511,8 @@ reference:
       title: docker import
     - path: /engine/reference/commandline/info/
       title: docker info
+    - path: /engine/reference/commandline/init/
```

Comment:
I don't think this should live in the reference docs. This page is not a reference but a set of examples. I think it should go in Manuals instead, like other plugins (e.g., https://docs.docker.com/scout/). cc @thaJeztah

Reply:
Hmm.. yes, I agree.

Wondering if the plugin can generate yamldocs (as we do for the other plugins?) @rafd123  ?

Reply:
Is this what you're referring to, @thaJeztah?

https://github.com/docker/docker-init/tree/main/docs/reference

Reply:
Thanks all.
@rafd123, yes, I think it's those files. Can we add the more comprehensive description and examples to those? If it's just a manually adding it to the yaml rather than some auto-generated thing, let me know and I can create a PR to update the yaml in the init repo.

@crazy-max, I'll remove the md files and use the yaml files to generate the content. I'm not sure what you mean by it's just a set of examples because the other reference docs have exactly the same sections. If you just mean it needs to be generated by the yaml, I'll do that, otherwise can you elaborate?

As for adding it to Manuals, we decided against it for this release as it is limited.

Since this goes live on Monday, I plan to just merge this PR to avoid any delays. I'll create a follow-up PR to update it to the yaml file. If any concerns, let me know.